P1/style.css

```diff
@@ -0,0 +1,176 @@
+BODY {
```

Comment:
just tips for the future. take a look at some best practices. for example http://cssreset.com/what-is-a-css-reset/